site/app/templates/polls/ViewPoll.twig

```diff
@@ -169,7 +169,7 @@
             [window.getComputedStyle(document.body).getPropertyValue('--histogram-correct'),
                 window.getComputedStyle(document.body).getPropertyValue('--histogram-default')];
             {% for option in poll.getOptions() %}
-                data[0].x.push("{{ option.getResponse()|replace({"\n": " ", "\r": " ", "\t": " "}) }}");
+                data[0].x.push("{{ option.getResponse()|e('js')|replace({"\n": " ", "\r": " ", "\t": " "}) }}");
```

Comment:
The `e('js')` should actually be at the end to ensure the replace happens on the string itself not the encoded string